recipes/nilearn/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "0.2.4" %}
+
+package:
+  name: nilearn
+  version: {{ version }}
+
+source:
+  fn: nilearn-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/n/nilearn/nilearn-{{ version }}.tar.gz
+  md5: 66145792fb2aebf51e221225bd210534
+
+build:
+    number: 0
+    script: python setup.py install
+
+requirements:
+  build:
+    - python
+    - numpy
+    - scipy
+    - scikit-learn
+    - nibabel
+  run:
+    - python
+    - numpy
+    - scipy
+    - scikit-learn
+    - nibabel
+    - matplotlib
+
+test:
+  imports:
+    - nilearn
+
+about:
+  home: https://nilearn.github.io
+  license: New BSD
```

Comment:
I never know if the New BSD is `BSD 2-Clause` or `BSD 3-Clause`. Do you mind updating that to the right version of the licence to make it easy to recognize?